spec/arrays.dd

```diff
@@ -181,8 +184,11 @@ p = a.ptr; // p points to the first element of the array a.
 
 // error, since the length of the array pointed to by p is unknown
 //s = p;
-//s = a; // error, as a's length is not known at compile-time
+
+s = [1,2]; // OK
+assert(s == [1,2,0]);
 s = [1,2,3];
+//s = [1,2,3,4]; // error, too many elements
```

Comment:
Please put spaces after commas

Reply:
Turns out assigning a shorter literal is a runtime error. I have now documented that and assigning a dynamic array variable.